dts/bindings/opamp/nxp,opamp.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+include:
+  - opamp-controller.yaml
+  - pinctrl-device.yaml
+
+compatible: "nxp,opamp"
+
+description: |
+  NXP Operational Amplifier (opamp)
+
+properties:
+  non-inverting-reference-source:
+    type: string
+    enum:
+      - "Input0"
+      - "Input1"
+      - "Input2"
+      - "Input3"
+    default: "Input1"
+    description: |
+      Selects non-inverting path reference voltage source.
+
+  mode:
+    type: string
+    enum:
+      - "low_noise"
+      - "high_speed"
+    default: "low_noise"
```

Comment:
all default values in DT bindings need justifications/explanations in the description